akka-stream-tests/src/test/scala/akka/stream/scaladsl/MergePreferredSpec.scala

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.scaladsl
+
+import akka.stream._
+import akka.stream.testkit.TwoStreamsSetup
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class MergePreferredSpec extends TwoStreamsSetup {
```

Comment:
Made a separate file, but would live together with the GraphMergePreferredSpec I guess